functions/config/development.json

```diff
@@ -2,5 +2,9 @@
     "DEBUG" : {
         "0" : "app:startup",
         "1" : "app:http"
-    }
+    },
+
+    "create" : "created successfully",
+    "update" : "updated successfully",
+    "delete" : "deleted successfully"
```

Comment:
These should be part of constants file /common place as they dont change based on environment.